doc/rst/dfilemaker.1.rst

```diff
@@ -48,7 +37,7 @@ OPTIONS
    ``alternate`` will fill the file with a 0xAA pattern. The default
    fill is ``random``.
 
-.. option:: -r, --ratio=*min*-*max*
+.. option:: -r, --ratio=*min*-*max*  (not yet implemented)
 
```

Comment:
If not yet implemented, might be safer not to show it at all as might never be implemented. My 2cents.

Reply:
Fixed